javascript/react-props_greeting/src/App.js

```diff
@@ -1,5 +1,23 @@
 import "./styles.css";
 
 export default function App() {
-  return <h1>Replace me with your component!</h1>;
+  return <Greeting name="Marcell" />;
 }
+function Greeting({ name }) {
+  if (name === "Marcell") {
```

Comment:
I thought, if-statements don't work in react? Did you try? :D If it doesn't, try a ternary statement.